databases/week2/createDatabaseBusiness.sql

```diff
@@ -0,0 +1,145 @@
+CREATE DATABASE business;
+Use business;
+CREATE TABLE `staff` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `duty` varchar(255) NOT NULL,
+  `status_id` int(10) unsigned NOT NULL,
+  CONSTRAINT `fk_status` FOREIGN KEY (`status_id`) REFERENCES `status` (`id`) ON DELETE CASCADE ON UPDATE CASCADE PRIMARY KEY (`id`)
+);
+CREATE TABLE `jobTitle` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `jobTitle` varchar(255) NOT NULL,
```

Comment:
Since the table is called _JobTitle_ you can consider naming this property _Title_ since it can be understood what it refers to

Reply:
sure.. thanks